react-native-libraries.json

```
@@ -16453,5 +16453,15 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/kingstinct/react-native-activity-kit",
```

Hey @efstathiosntonas, thanks for adding a new entry to the directory! 👍 

To fix the CI check we need to alter the GitHub URL of the entry and point it to the folder when package code resides:

```suggestion
    "githubUrl": "https://github.com/kingstinct/react-native-activity-kit/tree/main/packages/react-native-activity-kit",
```